content/news/public-oss-demo/2023/upcoming-live-update-1-2023.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title:  "Announcing January PrestaShop Project Live Update"
+subtitle: "Let's meet for the first Live Update in 2023"
+date:   2023-01-06 09:00:00
+authors: [ kpodemski ]
+icon: icon-video-camera
+tags: [contribute, event, community]
+---
+
+Dear community,
+
+The first Live Update for the PrestaShop Project will happen on [**Wednesday, January 11th, 2023 at 4pm CET**](https://www.youtube.com/watch?v=YDS51o48W1I)!
```

Comment:
For timezones, consider using time.is: https://time.is/1600_11_Jan_2023_in_Paris

Reply:
good idea, done